catalyst-gateway/bin/src/service/api/cardano/cip36/filter.rs

```diff
@@ -493,29 +492,43 @@ async fn get_all_invalid_registrations(
             continue;
         };
 
+        let payment_addr = match Cip19ShelleyAddress::try_from(row.payment_address) {
+            Ok(payment_addr) => Some(payment_addr),
+            Err(_) => None,
+        };
```

Comment:
Nitpick: matching here and below can be replaced by this:
```suggestion
        let payment_addr = Cip19ShelleyAddress::try_from(row.payment_address).ok();
```

Reply:
good catch, thanks!